tests/e2e/tests/i18n/extract-default.ts

```diff
@@ -0,0 +1,19 @@
+import {join} from 'path';
+import {ng} from '../../utils/process';
+import {
+  expectFileToExist, deleteFile, writeFile,
+  expectFileToMatch
+} from '../../utils/fs';
+
+
+export default function() {
+  const testComponentDir = join('src/app', 'i18n-test');
+  return ng('generate', 'component', 'i18n-test')
+    .then(() => deleteFile(join(testComponentDir, 'i18n-test.component.html')))
```

Comment:
You don't need to delete it, since you're overwriting it.  Also, you don't need to keep a variable for the path.